app/models/order.rb

```diff
@@ -10,4 +11,15 @@ def can_employee_afford_price
 
     errors.add(:can_not_afford, 'You can not afford this reward')
   end
+
+  def self.to_csv
+    attributes = %w[employee_id reward_id purchase_price status created_at updated_at]
```

Comment:
Good idea, but instead of hardcoding attirbutes that you would need to change every time you can use [attribute_names method](https://api.rubyonrails.org/classes/ActiveModel/Attributes/ClassMethods.html#method-i-attribute_names) like so `Order.attribute_names`, this way they will always be correct.

Reply:
Actually this is common practice, to define such attributes in the class-  I would only suggest to move them to the IMPORT_ATTRIBUTES constant